README.md

```diff
@@ -1,34 +1,33 @@
 
-## General
+The Intel Data Center GPU driver container image is built from the [Intel GPU Repository](https://github.com/intel-gpu). It includes:
+- [Intel® Graphics Driver Backports for Linux* OS (i915) for Red Hat Enterprise Linux (RHEL)](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/redhat/main) 
+- [Intel® Converged Security Engine (Intel® CSE) Backports](https://github.com/intel-gpu/intel-gpu-cse-backports/tree/main)
+- [Intel® Platform Monitoring Technology (Intel® PMT) Backports](https://github.com/intel-gpu/intel-gpu-pmt-backports/tree/main)
+- [Intel® GPU firmware](https://github.com/intel-gpu/intel-gpu-firmware)
 
-**Intel Data Center GPU Driver for OpenShift project** is used to build, package, certify and release Intel data center GPU driver container images for [Red Hat OpenShift Container Platform (OCP)](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform). Users can make use of the pre-built driver container image to provision Intel® Data Center GPU cards on OCP. They can also use [Intel data center GPU driver Dockerfile](docker/intel-dgpu-driver.Dockerfile) released from this project as a reference to on-premise build the driver container images and provision the GPU cards. Currently, [Intel® Data Center GPU Flex Series](https://www.intel.com/content/www/us/en/products/docs/discrete-gpus/data-center-gpu/flex-series/overview.html) is supported. [Intel® Data Center GPU Max Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/max-series.html) support is planned in a future release.
+# Install Intel Data Center GPU Driver on RHOCP
+We recommend users use the [Kernel Module Management (KMM) operator](https://docs.openshift.com/container-platform/4.12/hardware_enablement/kmm-kernel-module-management.html) to install and manage the Intel Data Center GPU driver on RHOCP. The KMM operator can be used to deploy all the necessary driver components as well as the firmware from within the driver container image. 
 
-Intel data center GPU driver container image is built from the [Intel GPU Driver Repository](https://github.com/intel-gpu). It includes:
--	[Intel® Graphics (i915) Driver Backports for Linux for RHEL](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/redhat/main)
--	[Intel® Converged Security Engine (CSE) Backports](https://github.com/intel-gpu/intel-gpu-cse-backports/tree/main)
--	[Intel® Platform Monitoring Technology (PMT) Backports](https://github.com/intel-gpu/intel-gpu-pmt-backports/tree/main)
--	[Intel® GPU firmware](https://github.com/intel-gpu/intel-gpu-firmware)
+To install Intel Data Center GPU drivers on OpenShift using the KMM operator, please follow [pre-build mode support](https://github.com/intel/intel-technology-enabling-for-openshift/tree/main/kmmo#managing-intel-dgpu-driver-with-kmm-operator) from the [Intel Technology Enabling for OpenShift](https://github.com/intel/intel-technology-enabling-for-openshift). 
```

Comment:
Just a kind reminder. In the future HTML based document, this link should point to the Intel Technology Enabling for OpenShift release readme doc.  Since this readme is in main branch - the development branch. It is OK to point to the main branch of Intel Technology Enabling for OpenShift project.